dataRetriever/mock/whiteListHandlerStub.go

```diff
@@ -2,31 +2,28 @@ package mock
 
 import "github.com/ElrondNetwork/elrond-go/process"
 
+// WhiteListHandlerStub -
 type WhiteListHandlerStub struct {
-	RemoveCalled            func(keys [][]byte)
-	AddCalled               func(keys [][]byte)
-	IsForCurrentShardCalled func(interceptedData process.InterceptedData) bool
-}
-
-func (w *WhiteListHandlerStub) IsForCurrentShard(interceptedData process.InterceptedData) bool {
-	if w.IsForCurrentShardCalled != nil {
-		return w.IsForCurrentShardCalled(interceptedData)
-	}
-	return true
+	RemoveCalled                 func(keys [][]byte)
+	AddCalled                    func(keys [][]byte)
+	IsWhiteListedWithResetCalled func(interceptedData process.InterceptedData) bool
```

Comment:
Not need to define its mock functionality? Does this stub implements any interface?

Reply:
yes, remained from old iteration. :+1: